contracts/HubPool.sol

```diff
@@ -80,43 +85,51 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
     event WhitelistRoute(address originToken, uint256 destinationChainId, address destinationToken);
 
     event InitiateRefundRequested(
-        uint64 indexed relayerRefundId,
         uint64 requestExpirationTimestamp,
-        uint64 poolRebalanceLeafCount,
+        uint64 unclaimedPoolRebalanceLeafCount,
         uint256[] bundleEvaluationBlockNumbers,
         bytes32 poolRebalanceRoot,
         bytes32 destinationDistributionRoot,
         address indexed proposer
     );
     event RelayerRefundExecuted(uint256 relayerRefundId, MerkleLib.PoolRebalance poolRebalance, address caller);
 
-    event RelayerRefundDisputed(uint256 relayerRefundId, address disputer);
+    event RelayerRefundDisputed(
+        address disputer,
+        SkinnyOptimisticOracleInterface.Request ooPriceRequest,
+        RefundRequest refundRequest
```

Comment:
One minor comment on these: we might want to select out parts of these structs we want to emit to 1) reduce the data and 2) make etherscan better able to encode/decode the events rather than just showing bytes.

Reply:
this is a good point and i'll change the same for SpokePool

Reply:
I'm going to leave this for now and will try and see how much of a gas difference this makes.

Reply:
i don't think it should change anything